<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmRecordReader</class>
 <widget class="QWidget" name="FrmRecordReader">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>203</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Raw Stream Reader</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item row="1" column="0">
    <spacer name="spacer6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="spacer4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout">
     <item row="1" column="0">
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="3">
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="textLabel1_">
         <property name="text">
          <string>Raw Stream Binary</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KUrlRequester" name="urlBinRec" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="3">
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel3">
           <property name="text">
            <string>Position</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edTime">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <spacer name="spacer3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>88</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout">
       <item>
        <layout class="QVBoxLayout">
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QPushButton" name="btnRW">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>RW</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnStop">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>STOP</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnFF">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>FF</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QPushButton" name="btnFirst">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>FIRST</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnBack">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>BACK</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnNext">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>NEXT</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Speed</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbSpeed">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>Fastest</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fast</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Slow</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Slowest</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>urlBinRec</tabstop>
  <tabstop>btnRW</tabstop>
  <tabstop>btnStop</tabstop>
  <tabstop>btnFF</tabstop>
  <tabstop>btnFirst</tabstop>
  <tabstop>btnBack</tabstop>
  <tabstop>btnNext</tabstop>
  <tabstop>cmbSpeed</tabstop>
  <tabstop>edTime</tabstop>
 </tabstops>
 <includes>
  <include location="local">kurlrequester.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
